tests/integration/targets/jenkins_credential/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+version: "3.8"
+
+services:
+  jenkins:
+    image: youssefkali/test-jenkins-credential-module:3.0 # Pre-configured Jenkins server
```

Comment:
Depending on how much configuration is being done there, it might be better to load a plain image and perform that configuration in here, so that you will have one less thing to maintain.